### PR TITLE
fix(content) make remnant cognizance 4 desc more helpful

### PR DESCRIPTION
**Pull request by OcelotWalrus**
# Bug Fix

This PR addresses the bug/feature described in issue #9967

## Summary
This PR simply changes the remnant cognizance 4 mission description from:

`Prefect Chilia has asked you to track down a Korath fleet before returning to <destination>.`
--> `Prefect Chilia has asked you to track down a Korath fleet detected in The Ember Waste before returning to <destination>.`, so that the description gives more info.

## Performance Impact

N/A

### CODE REVIEW

**Comment by OcelotWalrus** [comment]
> I am good with this, how does it look in-game?

Well like that, but with the new text
image

**Comment by brendanjones** [comment]
Why not also the change of mission intro text so the user doesn’t think the fleet will be in Parca?

From `According to our surveillance, that ship was part of a fleet that recently passed through Parca.` to `According to our surveillance, that ship was part of a fleet that recently passed through Parca, though the fleet will likely be on the move through The Ember Waste.`.

File: data/remnant/remnant 2 cognizance.txt [hunk 1]
```diff
@@ -143,7 +143,7 @@ mission "Remnant: Cognizance 3"
 mission "Remnant: Cognizance 4"
 	landing
 	name "Search for the Korath"
-	description "Prefect Chilia has asked you to track down a Korath fleet before returning to <destination>."
+	description "Prefect Chilia has asked you to track down a Korath fleet detected in The Ember Waste before returning to <destination>."
```
Comment (mOctave) [inline data/remnant/remnant 2 cognizance.txt:146]:
```suggestion
	description "Prefect Chilia has asked you to track down a Korath fleet detected in the Ember Waste before returning to <destination>."
```

Comment (OcelotWalrus) [reply data/remnant/remnant 2 cognizance.txt]: right.

**Review by mOctave — APPROVED** [review]

**Comment by warp-core** [comment]
> how does it look in-game?

image

**Pull request merged**